src/app/workflow/workflow-detail/workflow-detail.component.html

```diff
@@ -1,213 +1,300 @@
-    </button>
-    <button *ngIf="!workflow.publiclyShared && workflow.status === 'SUCCEEDED' && canEdit()" type="button"
-            ngbTooltip="Make workflow public"
-            placement="left"
-            class="btn btn-light" (click)="makeWorkflowPublic();">
-          <span class="fa fa-users"
-                aria-hidden="true"></span>
-    </button>
+<!--      <dt class="col-md-4">Owner:</dt>-->
+<!--      <dd class="col-md-8">{{workflow.owner}}</dd>-->
 
-  </div>
-</div>
+<!--      <dt class="col-md-4">Status:</dt>-->
+<!--      <dd class="col-md-8">{{workflow.status}}-->
+<!--        <span *ngIf="workflow.status === 'ERROR' && workflow.errorMessage"-->
+<!--            ngbTooltip={{workflow.errorMessage}}-->
+<!--            placement="right"-->
+<!--            aria-hidden="true"><i class="far fa-question-circle"></i>-->
+<!--        </span>-->
+<!--      </dd>-->
+<!--      <dt class="col-md-4">Visibility:</dt>-->
+<!--      <dd class="col-md-8">-->
+<!--        <div *ngIf="workflow.publiclyShared; else privateData">Public</div>-->
+<!--      </dd>-->
+<!--      <ng-template #privateData>Private</ng-template>-->
+<!--    </dl>-->
+<!--  </div>-->
+<!--  <div class="col-md-5">-->
+<!--    <button *ngIf="workflow.status === 'CREATED'"-->
+<!--            class="btn btn-outline-dark wipp-btn wf-btn-submit"-->
+<!--            [disabled]="jobs.length < 1" (click)="submitWorkflow()">-->
+<!--      <i class="fas fa-arrow-alt-circle-right">-->
+<!--        <p>Submit workflow</p>-->
+<!--      </i>-->
+<!--    </button>-->
+<!--    <button-->
+<!--      *ngIf="workflow.generatedName"-->
+<!--      class="btn btn-outline-dark wipp-btn wf-btn-submit">-->
+<!--      <i class="fas fa-sitemap"> </i>-->
+<!--      <a class="nav-link"-->
+<!--         href={{argoUiLink}}-->
+<!--         target="_blank"-->
+<!--         style="color:inherit; text-decoration: none; display: inline"-->
+<!--         [ngbTooltip]="argoUiTipContent"-->
+<!--      >-->
+<!--        <p style=" display: inline; font-family: 'Roboto', sans-serif ; font-weight: normal">-->
+<!--          Monitor in Argo-->
+<!--        </p>-->
+<!--      </a>-->
+<!--    </button>-->
+<!--  </div>-->
+
+<!--  <div class="col-md-2 text-right">-->
+
+<!--    <button *ngIf="canCreate()"-->
+<!--      class="btn btn-light" type="button"-->
+<!--      ngbTooltip="Copy workflow"-->
+<!--      placement="left"-->
+<!--      (click)="copyWorkflow()">-->
+<!--      <span class="fas fa-copy"-->
+<!--            aria-hidden="true"></span>-->
+<!--    </button>-->
+<!--    <button *ngIf="!workflow.publiclyShared && workflow.status === 'SUCCEEDED' && canEdit()" type="button"-->
+<!--            ngbTooltip="Make workflow public"-->
+<!--            placement="left"-->
+<!--            class="btn btn-light" (click)="makeWorkflowPublic();">-->
+<!--          <span class="fa fa-users"-->
+<!--                aria-hidden="true"></span>-->
+<!--    </button>-->
+
+<!--  </div>-->
+<!--</div>-->
```

Comment:
L92 to L151: Something to finish? Or maybe commented code can be remove right?

Reply:
Fixed